tests/Unit/ProtoTests/BasicDiregapic/out/src/resources/library_service_rest_client_config.php

```diff
@@ -48,6 +48,9 @@
                         'method' => 'post',
                         'uriTemplate' => '/v1/{name=bookShelves/*}/books',
                         'body' => 'book',
+                        'queryParams' => [
+                            'book',
```

Comment:
Given the body is `book`, is this correct?

Reply:
Fixed.